src/templates/typescript/typeAliases/typeAliasTemplate.ts

```diff
@@ -0,0 +1,17 @@
+import Handlebars from "../registerPartials";
+
+export const source = `{{~# unless isRootType ~}}
+import I{{ baseTypeName }} from "../{{ this.namespace }}/I{{ baseTypeName }}";
+{{/ unless }}
+type I{{ name }} = {{# unless isRootType }}I{{/ unless}}{{ baseTypeName }};
+export default I{{ name }}; 
+`;
+
+export interface ITemplateContext {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5648116)